cpp/src/community/k_truss_impl.cuh

```diff
@@ -41,8 +42,68 @@
 #include <thrust/transform.h>
 #include <thrust/tuple.h>
 
+#include <chrono>
```

Comment:
Is this used?

Rather not have dangling using statements like this in a header.  If we're actually using `std::chrono`, I think it's better to prefix the function calls/variable references (e.g. `std::chrono::duration`) rather than a using which loses the context of where `duration` comes from.

If this is just from some debugging/benchmarking that's not in use, best not to include it in the final code.

Reply:
> If this is just from some debugging/benchmarking that's not in use, best not to include it in the final code.

Yes those were only for benchmarking purposes and I removed them